packages/office-ui-fabric-react/src/components/Check/Check.base.tsx

```diff
@@ -14,7 +14,11 @@ export class CheckBase extends BaseComponent<ICheckProps, {}> {
   };
 
   public shouldComponentUpdate(newProps: ICheckProps): boolean {
-    return this.props.checked !== newProps.checked;
+    return (
+      this.props.checked !== newProps.checked ||
+      this.props.theme !== newProps.theme ||
```

Comment:
if i say:
loadTheme(foo);
loadTheme(foo);
will this still trigger? (i.e. does customizer copy or send through the original instance?)
not a big deal, super edge case anyways

Reply:
loadTheme would create a new theme object. so second would trigger.